pkg/volumemanager/persistent_volume_manager.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volumemanager
```

Comment:
Sure.  While I'm at it, name change?

I originally thought I'd need to sync 2 objects.  I don't, just claims for binding.  The custom store that indexes and sorts volumes is sufficient and the reflector keeps that cache populated.

Is this control loop just a ```PersistentVolumeClaimBinder``` ?